### PR TITLE
Log prompts

### PR DESCRIPTION
**Pull request by markVaykhansky**

### CODE REVIEW

File: token_benchmark_ray.py [hunk 1]
```diff
@@ -459,6 +467,15 @@ def run_token_benchmark(
         "name=foo,bar=1. These will be added to the metadata field of the results. "
     ),
 )
+args.add_argument(
+    "--log-prompts",
+    type=bool,
+    default=False,
+    help=(
+        "If True will log all prompts send to the model"
```
Comment (jonzarecki) [inline token_benchmark_ray.py]:
```suggestion
        "If True will log all prompts sent to the model"
```

File: token_benchmark_ray.py [hunk 2]
```diff
@@ -90,6 +91,11 @@ def get_token_throughput_latencies(
             prompt_tokens_stddev=stddev_input_tokens,
             expect_output_tokens=num_output_tokens,
         ))
+
+    if log_prompts:
+        print("Sending the following prompts:")
+        print(prompts)
```
Comment (jonzarecki) [inline token_benchmark_ray.py:97]: is it better to print one prompt-per-line? what will be easier to look at

Comment (jonzarecki) [reply token_benchmark_ray.py:97]: didn't you want to log the prompt length in tokens? much more interesting.

I imagine:
per-line
${prompt len in tokens} - $prompt

Comment (jonzarecki) [reply token_benchmark_ray.py:97]: Also, maybe when logging=False just print the prompt lengths (in one line or a histogram of lengths in tokens)

File: token_benchmark_ray.py [hunk 3]
```diff
@@ -459,6 +467,15 @@ def run_token_benchmark(
         "name=foo,bar=1. These will be added to the metadata field of the results. "
     ),
 )
+args.add_argument(
+    "--log-prompts",
+    type=bool,
+    default=False,
```
Comment (jonzarecki) [inline token_benchmark_ray.py:477]: do you want the default to be false? 
we can only change this in the argo sensor

**Review by jonzarecki — CHANGES_REQUESTED** [review]

File: token_benchmark_ray.py (hunk 3 above)
Comment (markVaykhansky) [reply token_benchmark_ray.py:477]: Yes I think it's spammy by default

File: token_benchmark_ray.py (hunk 2 above)
Comment (markVaykhansky) [reply token_benchmark_ray.py:97]: Prompts is an array of tuples (prompt, token_count). 

I'll add the log of the tokens

File: token_benchmark_ray.py (hunk 3 above)
Comment (jonzarecki) [reply token_benchmark_ray.py:477]: So you don't want it to be spammy. alright

**Review by jonzarecki — APPROVED** [review]

**Pull request merged**